### DISCUSSION TITLE
[NPC Aggression Timer] Have a toggle to disable the "Requires Calibration" message

### DISCUSSION THREAD

**Discussion by etc597** [runelite/runelite#15078] [Q&A] [answered]
Sometimes I log in and the npc aggression timer needs calibration but I just wanna afk fish. I don't want to turn the plugin off since I like having it on but I would like being able to toggle or clear that message if I don't care about calibrating the timer (if I'm doing a skilling activity, for instance).

**Comment by Enriath** [comment, marked as answer]
If you don't want to see the calibration message, turn the plugin off; you don't gain anything by having it enabled and uncalibrated vs just having it off.

**Reply by Adam-** [reply]
We used to have an option for this, but too many people would disable the message and then get confused as to why the plugin wasn't working.

**Reply by etc597** [reply]
What I gain is not having to remember to re-enable the plugin when I do teleport to an area where I am going to fight stuff. It would be nice to have a toggle for that message appearing without having to enable/disable the plugin. Like even just a checkbox that prevents the message from showing up (even if it is needed) would be more than sufficient. I can draft up a pr if necessary

**Reply by Enriath** [reply]
If you're going to forget to turn the plugin back on, you're going to forget to re-calibrate the plugin if you remove the overlay.

We have dealt with so many people over the years who would just blindly enable the option, then complain the plugin isn't working. Really, it never should have been an option to remove the overlay in the first place, as what it's telling you is integral to the plugin functioning.

As such, being able to hide the overlay is not something that will go back into the client, **full stop**. Turning off the plugin does exactly what you want in exactly the same way as you'd want, whilst not having users confused as to why a plugin they enabled isn't working.

**Reply by etc597** [reply]
That’s valid and I respect that. Could we have a toggle to display a shorter message? Something like “Npc Timer needs Calibration”. I ask this since I play in fixed and the ui takes up a lot of screen space
